### DISCUSSION TITLE
基础转发表1.2.0配置不生效的问题

### DISCUSSION THREAD

**Discussion by lf2186** [bfenetworks/bfe#820] [Q&A (问题和回答)] [answered]
{
    "Version": "20210830150000",
    "ProductRule": {
        "example_product": [
            {
                "ClusterName": "cluster_example",
                "Cond": "default_t()"
            }
        ]
    },
    "BasicRule":{
      "example_product" : [
      {
        "Hostname": ["bfe01.example.com"],
        "Path":["/*"],
        "ClusterName": "cluster_path"
      }
    ]
    }
}

bfe1.2.0复制的测试用例的配置，修改以后，发现请求bfe01.example.com域名的服务只会落在默认的cluster_example，如果上面ProductRule的默认配置删除，会报错，FindLocation error[no rule match for this req] host[bfe01.example.com]，是基础转发表不支持吗？？？还是配置的不对，应该怎么配？

**Comment by daimg** [comment]
基础转发表功能尚未合入到1.2.0版本中。可以尝试拉取develop分支的最新代码，编译新版本的bfe后，再验证上述功能。

**Comment by lf2186** [comment, marked as answer]
好的，感谢！～
